actix-http/src/cookie/secure/private.rs

```diff
@@ -57,9 +58,14 @@ impl<'a> PrivateJar<'a> {
         let unsealed = open_in_place(&key, nonce, ad, 0, sealed)
             .map_err(|_| "invalid key/nonce/value: bad seal")?;
 
-        ::std::str::from_utf8(unsealed)
-            .map(|s| s.to_string())
-            .map_err(|_| "bad unsealed utf8")
+        if let Ok(unsealed_utf8) = ::std::str::from_utf8(unsealed) {
+            Ok(unsealed_utf8.to_string())
+        } else {
+            error!("Private cookie does not have utf8 content!");
```

Comment:
Please use single invocation of `error` macro.
There is no need to flow with multiple invocations (remember that logs have cost)